bin/split_contigs.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
missing APACHE licence

Reply:
added. And now I am thinking that I don't need this script, as I am chunking the fasta in the subworkflow jeje